tests/keras_tests/function_tests/test_tensorboard_writer.py

```diff
@@ -35,7 +35,7 @@ class TestLogger(unittest.TestCase):
     def setUpClass(cls):
         mct.core.common.Logger.set_log_file('/tmp/')
         model = MobileNet()
-        mct.keras_post_training_quantization(model, random_datagen, n_iter=1)
+        mct.keras_post_training_quantization(model, random_datagen, n_iter=1, analyze_similarity=True)
 
```

Comment:
Please use this flag in https://github.com/sony/model_optimization/blob/main/tests/pytorch_tests/function_tests/test_tensorboard_writer.py as well.